daprdocs/content/en/developing-applications/building-blocks/actors/namespaced-actors.md

```diff
@@ -0,0 +1,109 @@
+---
+type: docs
+title: "Namespaced actors"
+linkTitle: "Namespaced actors"
+weight: 40
+description: "Learn about namespaced actors"
+---
+
+Namespaced actors use the multi-tenant Placement service. With this service, in a scenario where each tenant has its own namespace, sidecars belonging to a tenant named "Tenant A" won't receive placement information for "Tenant B". 
```

Comment:
Update to the following put then place this at end of this article. This is an implementation detail and not that important. 

```suggestion
Namespaced actors use the multi-tenant Placement service. With this control plane service where each application deployment has its own namespace, sidecars belonging to an application in namespace "ActorA" won't receive placement information for an application in namespace "ActorB". 
```